src/pyfmi/fmi3.pyx

```diff
@@ -1941,6 +1973,201 @@ cdef class FMUModelBase3(FMI_BASE.ModelBase):
 
         return variable_dict
 
+    cpdef get_output_dependencies(self):
+        """
+        Retrieve the variables that the (continuous, float64) outputs are
+        dependent on. Returns two dictionaries, one with the states and
+        one with the (continuous, float64) inputs.
+        """
+        # TODO: More than just float64 outputs?
+        # caching
+        if (self._outputs_states_dependencies is not None and
+            self._outputs_inputs_dependencies is not None):
+               return self._outputs_states_dependencies, self._outputs_inputs_dependencies
+
+        cdef FMIL3.fmi3_import_variable_t      *variable
```

Comment:
This one is quite different from FMI2, since we changed the FMIL API to dependencies.